plugins/dependencies/src/main/kotlin/dependencies/Testing.kt

```diff
@@ -255,12 +253,7 @@ object Testing : IsNotADependency {
         val inline = module("mockito-inline")
         val errorProne = module("mockito-errorprone")
         val junitJupiter = module("mockito-junit-jupiter")
-
-        /**
-         * Using Mockito with Kotlin
-         * [More info here](https://github.com/nhaarman/mockito-kotlin)
-         */
```

Comment:
I suppose that we don't need any additional comment since `mockito-kotlin` is part of offficial Mockito library now.